src/main/java/com/agileavengers/icuconnectbackend/repository/UserRepository.java

```diff
@@ -1,7 +1,11 @@
 package com.agileavengers.icuconnectbackend.repository;
 
-import com.agileavengers.icuconnectbackend.model.User;
 import org.springframework.data.jpa.repository.JpaRepository;
 
+import com.agileavengers.icuconnectbackend.model.User;
+
 public interface UserRepository extends JpaRepository<User, Long> {
+    User findByUsername(String username);
```

Comment:
Maybe you could use Optional<User> to ensure that if it is empty, it cannot cause a Null Pointer?

Reply:
Noted, made the change